packages/strickland/src/strickland.js

```diff
@@ -0,0 +1,45 @@
+import every from './every';
+import props from './props';
+
+export default function validate(rules, value, validateProps) {
+    let result = true;
+
+    if (Array.isArray(rules)) {
+        rules = every(rules);
+    } else if (typeof rules === 'object' && rules) {
```

Comment:
`typeof rules === 'object'` will not always produce the expected result:  https://jsfiddle.net/s1ygo1rn/

Reply:
I *think* I'm OK with this not being too terribly defensive, but what would you suggest?